### DISCUSSION TITLE
How change the form layout on outside filement form(landing page)

### DISCUSSION THREAD

**Discussion by harshaninv** [filamentphp/filament#13544] [Help]
### Package

Other

### Package Version

v3.0.0

### How can we help you?

Can someone know how to change the form layout on outside filament form(landing page):
    public function form(Form $form): Form
    {
        return $form
            ->schema([
                Fieldset::make('Basic Information')
                    ->schema([
                        TextInput::make('public_reporter_name')
                            ->label('Your Name')
                            ->required(),
                        TextInput::make('contact_number')
                            ->label('Phone Number')
                            ->required(),
                        TextInput::make('nic_id')
                            ->label('NIC ID')
                            ->required(),
                    ])
                    ->columns(3),
                Section::make('Report Details')->schema([
                    Grid::make(2)->schema([
                        Select::make('district_id')
                            ->label('Select the District')
                            ->options(function (callable $get) {
                                return District::all()->pluck('name', 'id');
                            })
                            ->live(),
                            
this grid and column layout does not work in out side forms.. what is the issue?

**Comment by leandrocfe** [comment]
Could you share an image?
